pkg/ceph/client/status.go

```diff
@@ -59,7 +59,9 @@ type HealthStatus struct {
 
 type CheckMessage struct {
 	Severity string `json:"severity"`
-	Message  string `json:"message"`
+	Summary  struct {
```

Comment:
this is a backport from what's in master, so there won't be any changes here.  However, this is a very common pattern i've seen when the struct is being used to serialize/deserialize JSON.  It helps a lot with readability to see the struct inline as it looks like in the JSON.  If the struct isn't used independently anywhere else, embedding it works very well IMO.

Here's another example of where we're using embedded structs with JSON serialization to improve readability: https://github.com/rook/rook/blob/master/pkg/ceph/client/osd.go#L28

note that i'm going to close this and re-open a new one because i opened it from the wrong branch.